tests/test_bgc_prokka.nf.test

```diff
@@ -0,0 +1,45 @@
+nextflow_pipeline {
+
+    name "Test pipeline: NFCORE_FUNCSCAN"
+    script "main.nf"
+    tag "pipeline"
+    tag "nfcore_funcscan"
+    tag "test_bgc_prokka"
+
+    test("test_bgc_prokka_profile") {
+
+        when {
+            params {
+                outdir = "$outputDir"
+            }
+        }
+
+        then {
+            assertAll(
+                { assert workflow.success },
+                { assert new File("$outputDir/pipeline_info/nf_core_pipeline_software_mqc_versions.yml").exists() },
+                { assert new File("$outputDir/multiqc/multiqc_report.html").exists() },
+
+                // antiSMASH
+                { assert path("$outputDir/bgc/antismash/sample_2/sample_2.gbk").text.contains("##antiSMASH-Data-START##") },                  // channel: gbk_input
+                { assert snapshot(path("$outputDir/bgc/antismash/sample_2/css")).match("antismash_css") },                                    // parts of channel: html_accessory_files
+                { assert path("$outputDir/bgc/antismash/sample_2/sample_2.zip").exists() },                                                   // channel: zip
+                { assert path("$outputDir/bgc/antismash/sample_2/index.html").text.contains("https://antismash.secondarymetabolites.org/") }, // channel: html
```

Comment:
For asserting for `text.contains`, are u sure this is working properly in this case? Because in its snap i dont find any 'true'/'false' for these assertions ? When i look at the test_full, u checkfor text and its snap u get a bunch of 'True' for these assertions

Reply:
The only diff i see is that here u assert only and in the full test you create a snapshot, is there a reason for that ?

Reply:
Yeah, if the e.g. contains or exists is not snapshot it does get checked but isn't listed. I've already noted that via Jasmin [here](https://github.com/nf-core/funcscan/issues/398), as it's incosnsitent but it does still 'functionally' work.